src/components/shared/FileUpload/SingleFileUpload.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { ComponentType, useImperativeHandle } from 'react';
+import classnames from 'classnames';
+
+import { withForwardingRef, ForwardedRefProps } from '~utils/hoc';
+import useDropzoneWithFileReader from '~hooks/useDropzoneWithFileReader';
+import { SingleFileUploadProps } from './types';
+
+import styles from './SingleFileUpload.css';
+
+const displayName = 'SingleFileUpload';
+
+const SingleFileUpload = ({
```

Comment:
I believe this is easier to follow than using the `FileUpload` component, which is tailored more for the CSV uploader than the Avatar uploader.